frontend/src/static/js/v_authorship.js

```diff
@@ -315,10 +323,11 @@ window.vAuthorship = {
     },
   },
 
-  created() {
+  mounted() {
     this.initiate();
     this.setInfoHash();
   },
+
```

Comment:
This line is added to follow the new line standard. Actually, we do not have a convention on when to have a new line in between the methods.